### DISCUSSION TITLE
Add a cross platform library to add a Badge to an App Icon

### DISCUSSION THREAD

**Discussion by bernmuel** [xamarin/XamarinCommunityToolkit#1549] [Ideas]
Are you planning to add a library to add a Badge to an App Icon?

**Comment by pictos** [comment]
Hey, @bernmuel I believe that's out of our roadmap. I believe that it fits better in Xamarin.Essentials

**Reply by bernmuel** [reply]
Thanks for the feedback.  I'm wondering if the Shiny library may add support for this.

**Reply by bernmuel** [reply]
I'll check there
